packages/inter-protocol/src/proposals/replaceElectorate.js

```diff
@@ -0,0 +1,326 @@
+// @ts-check
```

Comment:
Consider adding a `/** @file ... */` comment with a line or two of explanation and a reference to `replace-electorate-core.js`.